zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -9305,6 +9320,51 @@ components:
 
             For now, however, the start instruction is implicitly a "startBeforeElement" instruction
           type: string
+    RuntimeInstructionType:
+      type: string
+      description: |
+        The type of the runtime instruction. For now, only SUSPEND_PROCESS_INSTANCE is supported.
+
+        Caution: process instance suspension is an alpha feature and may be subject to change
+        in future releases.
+
+        Note the following limitations of suspended process instances:
+          - Suspended process instances currently cannot be resumed. The only possible state
+            transition is to cancel the suspended instance.
+          - Process instances suspended with the current implementation may not be fully recoverable
+            in the future, when the resume functionality is implemented.
```

Comment:
🔧 Let's expand this by stating that all jobs, user tasks, incidents, and event subscriptions are removed from the runtime state in the current implementation.